frontend/packages/dev-console/integration-tests/support/step-definitions/web-terminal/web-teminal-basic.ts

```diff
@@ -0,0 +1,30 @@
+import { When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { webTerminalPage } from '../../pages/web-terminal/webTerminal-page';
+
+When('user clicks on Open Terminal in new tab button on the terminal window', () => {
+  webTerminalPage.verifyOpenInNewTabButton();
+});
+And('user close current Web Terminal session', () => {
```

Comment:
Can you please replace `And` with either `Given`, `When` or `Then`?

Reply:
Replace on Then